nexus/src/nexus.rs

```diff
@@ -1698,23 +1704,26 @@ impl Nexus {
             ipv6_block,
         );
 
-        // Create the subnet record in the database. Overlapping IP ranges should be translated
-        // into an internal error. That implies that there's already an existing VPC Subnet, but
-        // we're explicitly creating the _first_ VPC in the project. Something is wrong, and likely
-        // a bug in our code.
+        // Create the subnet record in the database. Overlapping IP ranges
+        // should be translated into an internal error. That implies that
+        // there's already an existing VPC Subnet, but we're explicitly creating
+        // the _first_ VPC in the project. Something is wrong, and likely a bug
+        // in our code.
```

Comment:
Ahh you folks with your narrow terminals :)

Joking aside, I'm actually curious about this. My editor seems to wrap at 100 columns, which I should change, but is "ok". However, we have a `max_width = 80` line in our `rustfmt.toml`. I don't understand why that doesn't get applied to things like this when running `cargo fmt`.

Reply:
Yeah, the intent is to wrap at 80 columns for everything.  rustfmt only does this if you set [`wrap_comments`](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=wrap_comments#wrap_comments), which is unstable.